recipes/tests.rb

```diff
@@ -593,3 +593,22 @@ module load armpl/#{node['cfncluster']['armpl']['version']}
     user node['cfncluster']['cfn_cluster_user']
   end
 end
+
+###################
+# Pcluster AWSBatch CLI
+###################
+if node['cfncluster']['cfn_scheduler'] == 'awsbatch'
```

Comment:
minor: ideally there would be another condition here that prevents this block from running during the compute fleet kitchen tests. in practice i don't think it will matter since we don't run the kitchen tests with batch for compute fleets.